### ISSUE TITLE
The calendar does not work with personium-core 1.7.18b

### ISSUE THREAD

**Issue by tochi-y** [personium/app-personium-calendar#155]
This error message is displayed in the screen.

<img src="https://user-images.githubusercontent.com/32325140/66016196-985d4e80-e510-11e9-9fa3-7ad3d526ebdf.png" width="320px"/>

### The http acceess

GET https://test.example/test_app-personium-calendar/Engine/setAccessInfo

Statuscode: 403
Responsebody:
```json
{"code":"PR403-AU-0002","message":{"lang":"en","value":"Necessary privilege is lacking."}}
```

**Comment by shimono** [comment]
Could you give us a bit more detailed info?
Is it failing engine script execution ? 
Or the executed script failed to call Personium API?

**Comment by tochi-y** [comment]
@shimono I tested in the environment replaced with v1.7.18b that run in v1.7.17 before. So cell owner value may starts with `https://`. To avoid this affect, I created a new environment with v1.7.19 and did a same test. Then the result changed.

The app
1. ✅ launched via home app.
1. ✅ Executed Engine/start_oauth2.js. The response is 303.
1. ✅ Redirected to Personium Authorization Endpoint(__authz). The response is 303.
1. ✅ Redirected to execute Engine/receive_redirect.js with code. The response is 200.
    1. ✅ got  App token.
    1. ✅ got  App auth user token (called Personium token Endpoint(__token)).
1. ✅ Got box URL with access token. (called Personium Retrieve BoxURL  API (__box)).
1. ❌ Failed to get the box meta data. (called Personium Retrieve Box Meta Data API (calendar/)).

### Request

GET https://tochi.personium.example/calendar/
Authorization: Bearer AV~wSg*******xWA

### Response

Status Code: 403 

Body:

```javascript
{
  code: "PR403-AU-0002",
  message: {
    lang: "en",
    value: "Necessary privilege is lacking."
  }
}
```

**Comment by tochi-y** [comment]
@shimono I attached personium-core.log. Please see it to investigate.

```log
2019-10-23 11:34:06.224 [catalina-exec-47] [INFO ] PersoniumCoreContainerFilter [1.7.19] Started. GET http://tochi.personium.example/personium-core/calendar/ *.*.*.*
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter == Reqeust Headers
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter x-forwarded-for:*.*.*.*
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter x-forwarded-proto:https
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter x-forwarded-path:/calendar/
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter host:tochi.personium.example
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter connection:close
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter pragma:no-cache
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter cache-control:no-cache
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter accept:application/json
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter authorization:Bearer AV~5onX********Cy1PJ_Q
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter user-agent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.120 Safari/537.36
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter sec-fetch-mode:cors
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter origin:https://app-personium-calendar.personium.example
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter sec-fetch-site:same-site
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter referer:https://app-personium-calendar.personium.example/__/html/app.html?lng=ja
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter accept-encoding:gzip, deflate, br
2019-10-23 11:34:06.224 [catalina-exec-47] [DEBUG] PersoniumCoreContainerFilter accept-language:ja,en-US;q=0.9,en;q=0.8
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource Call API 'https://tochi.personium.example/calendar/'.
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     p_cookie: null
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     p_cookie_peer: null
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     Authorization: Bearer AV~5onX********Cy1PJ_Q
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     X-Personium-Unit-User: null
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     X-Personium-RequestKey: null
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     X-Personium-EventId: null
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     X-Personium-RuleChain: null
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] FacadeResource     X-Personium-Via: null
2019-10-23 11:34:06.225 [catalina-exec-47] [INFO ] PersoniumCoreLog [PL-SV-0022] - RequestKey generated: [PCS-6b0aaafc-3349-48e8-944c-425f23434cfa] - [io.personium.core.rs.FacadeResource#facade:101]
2019-10-23 11:34:06.225 [catalina-exec-47] [DEBUG] AccessContext io.personium.common.auth.token.VisitorLocalAccessToken
2019-10-23 11:34:06.226 [catalina-exec-47] [DEBUG] DavMetadataFile load started.
2019-10-23 11:34:06.226 [catalina-exec-47] [DEBUG] PersoniumCoreLog [PL-DV-0005] - Start file operation. [path=/personium_nfs/personium-core/dav/u0_unitadmin/KVi3YOlzRhqx8X0mo3jwYA/.pmeta] - [io.personium.core.model.impl.fs.DavMetadataFile#doLoad:201]
2019-10-23 11:34:06.226 [catalina-exec-47] [DEBUG] PersoniumCoreLog [PL-DV-0006] - End file operation. [time=0ms, size=0KB] - [io.personium.core.model.impl.fs.DavMetadataFile#doLoad:211]
2019-10-23 11:34:06.226 [catalina-exec-47] [DEBUG] DavMetadataFile load ended.
2019-10-23 11:34:06.227 [catalina-exec-47] [DEBUG] PersoniumCoreLog [PL-ES-0002] - ESReq index=u0_unitadmin type=Role node=n3iaou_TRVSgoszx60jotw reqType=Get data=null - [io.personium.core.model.impl.es.EsModel$2#handleEvent:57]
2019-10-23 11:34:06.232 [catalina-exec-47] [DEBUG] DavCmpFsImpl ###n3iaou_TRVSgoszx60jotw:https://tochi.personium.example/__role/__/admin
2019-10-23 11:34:06.232 [catalina-exec-47] [DEBUG] CellKeys load started.
2019-10-23 11:34:06.232 [catalina-exec-47] [DEBUG] CellKeys load ended.
2019-10-23 11:34:06.233 [catalina-exec-47] [DEBUG] DavMetadataFile load started.
2019-10-23 11:34:06.233 [catalina-exec-47] [DEBUG] PersoniumCoreLog [PL-DV-0005] - Start file operation. [path=/personium_nfs/personium-core/dav/u0_unitadmin/KVi3YOlzRhqx8X0mo3jwYA/kLYCrVEnQLWX9MDeNUXmhQ/.pmeta] - [io.personium.core.model.impl.fs.DavMetadataFile#doLoad:201]
2019-10-23 11:34:06.234 [catalina-exec-47] [DEBUG] PersoniumCoreLog [PL-DV-0006] - End file operation. [time=0ms, size=0KB] - [io.personium.core.model.impl.fs.DavMetadataFile#doLoad:211]
2019-10-23 11:34:06.234 [catalina-exec-47] [DEBUG] DavMetadataFile load ended.
2019-10-23 11:34:06.234 [catalina-exec-47] [INFO ] PersoniumCoreExceptionMapper [PR403-AU-0002] - Necessary privilege is lacking.
2019-10-23 11:34:06.236 [catalina-exec-47] [INFO ] PersoniumCoreContainerFilter [1.7.19] Completed. 403 12ms
```

**Comment by tochi-y** [comment]
Resolved in personium-core v1.7.20.
